aws-android-sdk-core/src/main/java/com/amazonaws/logging/ApacheCommonsLogging.java

```diff
@@ -15,14 +15,14 @@
 
 package com.amazonaws.logging;
 
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
 import com.amazonaws.logging.LogFactory.Level;
 
 /**
  * This is a wrapper over the Apache Commons
  * Logging framework.
+ * @deprecated Use {@link AndroidLog} instead.
  */
+@Deprecated
 public class ApacheCommonsLogging implements com.amazonaws.logging.Log {
```

Comment:
Can we delete the file?

Reply:
Yea, I doubt any customer was using this explicitly... but still I feel iffy about downright deleting a public class